README.md

```diff
@@ -1 +1,30 @@
 # Hello-Microverse
+
+This is my first Microverse project consisting of a simple HTML and CSS file that outputs "Hello Microverse".
+
+# Built With
+
+HTML
+CSS
+
+# Key Features
+
+Prints out a heading that reads "Hello Microverse" in the colour aubergine.
+
+# Live Demo
+
+N/A
+
+# Authors
+
+Author: Aamir Asaram
+
+GitHub: @aamir-asaram
+Twitter: @aaaaamir_
+LinkedIn: 
+
+# Future Features
+
+Add images
+Add more text
+
```

Comment:
- [ ] **Good job so far 👏 👏** However, As suggested in the project requirement, It would be better to all the headers from the Microverse template to your README file content to make it professional and in order to get the exact preview as that of Microverse, you may copy get a copy from [this ](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md)raw link 😉, 

  Kindly, As suggested by the previous reviewer, Remember that all the **Header sections** are required except the FAQ, 👍